pkg/signer/request-signature-streaming.go

```
@@ -48,10 +48,14 @@ const (
 
 // Request headers to be ignored while calculating seed signature for
 // a request.
+//
+// Per AWS SigV4 specification:
+//   "If the Content-Type header is present in the request, you must add it
```

golangci-lint (2.13.2)

[error] File is not properly formatted

(gofumpt)